sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -176,11 +176,12 @@ private String prettyPrintIfNeeded(ClientLogger logger, String contentType, Stri
         return result;
     }
 
-    private long getContentLength(HttpHeaders headers) {
+    private long getContentLength(ClientLogger logger, HttpHeaders headers) {
         long contentLength = 0;
         try {
             contentLength = Long.parseLong(headers.getValue("content-length"));
         } catch (NumberFormatException | NullPointerException ignored) {
+            logger.warning("Http logging cannot parse the content-length: " + headers.getValue("content-length"));
```

Comment:
I would use placeholders so that evaluation of headers.getValue isn't performed if the log level is not warning. In this case, 
```java
logger.warning("HTTP logging cannot parse the content-length: {}", headers.getValue("content-length"));
```

https://www.slf4j.org/manual.html#typical_usage

Reply:
This isn't resolved.

Reply:
Missed it. Checked in my changes.